fixtures/src/components/controlled/date.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import Date from 'carbon-react/lib/__experimental__/components/date';
+import { LogConsumer } from '../log';
+
+const ControlledDate = () => {
+  const [state, setState] = useState('');
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = (e) => {
+          setState(e.target.value.rawValue);
+          log(e, { method: 'onChange' });
+        };
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <div id='controlled_date'>
+              <h1>Controlled Date</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be the
```

Comment:
Once #2385 (FE-2124) is merged, `e.target.value` will be a `{ formattedValue, rawValue }` object (instead of an ISO date string). Should that be mentioned here in the documentation?

Reply:
Thanks for updating this documentation in 3925e26 :white_check_mark: